deployment/utils.py

```diff
@@ -205,3 +207,30 @@ def sample_nodes(
 
     result = sorted(ursulas, key=lambda x: x.lower())
     return result
+
+
+def _split_into_sorted_pairs(lst) -> List[List[str]]:
+    """Splits a list into sorted pairs."""
+    if len(lst) % 2 != 0:
+        # TODO: Use a group of 3 instead of sampling a node twice
+        lst.append(lst[-2])  # Use the second-to-last element to fill
+    it = iter(lst)
+    return [sorted(list(pair)) for pair in zip_longest(it, it)]
```

Comment:
I have this suggestion to address the TODO comment:

https://github.com/KPrasch/nucypher-contracts/pull/4

Reply:
Thank you! ✔️